### DISCUSSION TITLE
Use other maps for terrian generation?

### DISCUSSION THREAD

**Discussion by Superstrike1201** [jetelain/ArmaRealMap#53] [General]
Is there any way to use another sat map source for terrain generation. I generated my hometown but because open street map has made my town and its surroundings treeless the generated map looks barren and bad.

**Comment by jetelain** [comment]
In theory, it could be feasible to connect to another map provider. But I'm unsure other providers will gives as much as information as OpenStreetMap can.
I have chosen OpenStreetMap because other providers are very costly to get terms that allow reuse, and I could not afford to pay for everyone use.
An option would be to contribute to OpenStreetMap to improve details on our area. It’s quite accessible with the web editor (called “iD”) : you have an aerial view, and then you have to point and click areas. I’ve planned to add a page in the wiki to explain which elements are used by the generator and how you can map them on OpenStreetMap.

**Reply by Superstrike1201** [reply]
Hey thanks for the reply. I have begun fixing it on open street map but its going to take me days to do lol. Very cool project btw!
